folium/features.py

```diff
@@ -708,6 +709,13 @@ def render(self, **kwargs):
         """Renders the HTML representation of the element."""
         if isinstance(self._parent, GeoJson):
             keys = tuple(self._parent.data['features'][0]['properties'].keys())
+            geom_collections = [feature['id'] for feature in self._parent.data['features'] if feature['geometry'][
+                'type'] == 'GeometryCollection']
+            if any(geom_collections):
+                warnings.warn(" ".join("""GeoJsonTooltip is not configured to render tooltips for GeoJson
```

Comment:
This splitting and joining is quite clever. But I'd prefer to keep it simple and just split the string over multiple lines. So we don't have to figure out what's going on with the joins and splits.
```
warnings.warn('this is a '
    '{} long string'
    .format(3.14))
```

Or 

```
warnings.warn(
    'this is a {} '
    'long string'
    .format(3.14)
)
```

Reply:
That's what I had done initially, but the line breaks in the string were introducing some strange spacing from the tabs and newline characters - so I had gone this route to try and clean up the output.

Reply:
I don't really understand what you are saying. The example I showed shouldn't lead to newline characters.

Reply:
I can't seem to replicate the behavior on the machine I'm on right now but when I was working on it at home last night, having multiple concatenated strings read into the warning prompt was introducing some strange character spacing. I thought I had tried both approaches, having multiple strings which were concatenated, and a multiline-string with triple-quotes, but it may have just been the latter. I'll refactor to this as your suggestion doesn't exhibit the same issues. Thanks!